files/usr/bin/captive-portal

```diff
@@ -4,12 +4,32 @@
 # TODO add a parallel catch-all uhttpd service that shows the captive portal locally
 
 # Cleanup
+echo "Starting captive portal service."
+echo "You'll need to configure the web server running within the network."
+read -p "What is the webservers ipv4 address: " ANYGW4
+read -p "What is the webservers ipv6 address: " ANYGW6
 
-ANYGW4=10.5.32.1
```

Comment:
You could add a default value as fallback to the previous version

`ANYGW4="${ANYGW4:-10.5.32.1}"`

Reply:
The default would be the ip of anygw, so you can fetch it from your uci config.
Also, this does not seem to be used anywhere.